src/util/error/OutputAmountExceedsAvailableLiquidityError.ts

```diff
@@ -0,0 +1,7 @@
+class OutputAmountExceedsAvailableLiquidityError extends Error {
+  constructor(message = "Output amount exceeds available liquidity") {
```

Comment:
I think this version is also fine but I'm just wondering is there a reason to create an custom error class, instead of just keeping the error message:
```tsx
const X_ERR_MESSAGE = "x happened"
//...

throw new Error(X_ERR_MESSAGE);
```